Android/app/src/main/java/org/opendroneid/android/app/DebugActivity.java

```diff
@@ -279,9 +296,11 @@ private void initialize() {
         mModel.setAllAircraft(dataManager.getAircraft());
 
         final Observer<Set<AircraftObject>> listObserver = airCrafts -> {
-            if (airCrafts == null)
-                return;
-            setTitle(String.format(Locale.US, "%d drones", airCrafts.size()));
+            if (airCrafts == null) {
+                setTitle(String.format(Locale.US, "Scanning...", airCrafts.size()));
+        } else {
+                setTitle(String.format(Locale.US, "%d Drones Detected", airCrafts.size()));
+            }
```

Comment:
As you stated in issue #62, this code doesn't work. Please remove it from this Pull Request so we are not merging code that doesn't work.